CMakeLists.txt

```diff
@@ -117,6 +117,9 @@ set(COMMON_LINKER_FLAGS "")
 if (MSVC)
     list(APPEND COMMON_COMPILER_FLAGS
         /MP
+        # printf warnings as errors
+        /we4477
+        /we4474
```

Comment:
do we want to move those to clean-core/cmake/CompileOptions instead?

Reply:
yup!